### PR TITLE
Add text label for airplay active state

### PR DESCRIPTION
**Pull request by BogdanBilonog**
Show `AirPlay is active` label for AirPlay active state. Hide label if content controls are shown.

32658322-16455618-c622-11e7-9704-0ed169abd823
32658323-169d9b7a-c622-11e7-94b4-7a980922e100

### CODE REVIEW

File: PlayerControls/sources/AdVideoControls.swift [hunk 1]
```diff
@@ -23,10 +23,13 @@ public final class AdVideoControls: UIViewController {
     
     @IBOutlet public weak var containerView: UIView!
     
+    @IBOutlet public weak var airplayActiveLabel: UILabel!
```
Comment (AndriiMoskvin) [inline PlayerControls/sources/AdVideoControls.swift]: If airplays is active we will show the on top of advertisement?

**Comment by AndriiMoskvin** [comment]
I prefer having design similar to the native iOS player. This looks not good. Probably we should contact  Sigma designer to improve this.

File: PlayerControls/sources/AdVideoControls.swift (hunk 1 above)
Comment (BogdanBilonog) [reply PlayerControls/sources/AdVideoControls.swift]: What do you mean "on top of advertisement"? Image casts to Apple TV, iPhone's screen is black and view will show on top of it.

**Comment by BogdanBilonog** [comment]
img_2403

**Comment by BogdanBilonog** [comment]
img_2410
img_2411

**Comment by AndriiMoskvin** [comment]
@BogdanBilonog Update screenshots in PR.

**Pull request merged**